### PR TITLE
Shows specific medication with drugs name, type, category, descriptio…

### PR DESCRIPTION
**Pull request by JPD99**
…n, freqnency, strength, and taken status. Yarn jest that looks for the drug category of Atorvastatin. Added a id key for mockMedication. Added the route path for the protected show page.

### CODE REVIEW

File: app/javascript/components/pages/ProtectedShow.js [hunk 1]
```diff
@@ -25,6 +26,7 @@ const ProtectedShow = ({
         {myOwnMedications?.map((DisplayMyMeds)=>{
         return (
               <>
+              <NavLink to={`/${DisplayMyMeds.user_id}/medications/${DisplayMyMeds.id}`}>
```
Comment (SunkissedQueen) [inline app/javascript/components/pages/ProtectedShow.js]: Please be mindful of the naming conventions of parameters. Javascript parameters should be camelCased. Referring to Line 26

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Please ensure proper naming conventions are being used. After you make the change, please ensure that the app is still working as well as the tests. Ping me when the changes are made.

File: app/javascript/components/pages/ProtectedShow.js (hunk 1 above)
Comment (JPD99) [reply app/javascript/components/pages/ProtectedShow.js]: Fixed it! The application is still working properly.

**Review by SunkissedQueen — APPROVED** [review]
Good catch

**Pull request merged**